dateutil/parser.py

```diff
@@ -712,6 +715,11 @@ def _parse(self, timestr, dayfirst=None, yearfirst=None, fuzzy=False,
                         if len_li == 4:
                             res.minute = int(s[2:])
 
+                    elif len_li == 5 and len_l == 1 and value_repr.count('.') == 1:
+                        after_split = value_repr.split('.')
```

Comment:
I don't think this is the right way to handle this. The tokenizer should be doing the splitting, we should try to avoid hard-coding tokens into the token handling logic.